subsys/net/lib/openthread/platform/openthread-core-zephyr-config.h

```diff
@@ -98,11 +98,10 @@
  * in platform.
  *
  */
-#ifdef CONFIG_OPENTHREAD_PLATFORM_USEC_TIMER_ENABLE
-#define OPENTHREAD_CONFIG_PLATFORM_USEC_TIMER_ENABLE 1
-#endif
+#define OPENTHREAD_CONFIG_PLATFORM_USEC_TIMER_ENABLE (OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE && \
+(OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2))
```

Comment:
Just a small style nit
```suggestion
#define OPENTHREAD_CONFIG_PLATFORM_USEC_TIMER_ENABLE                           \
       (OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE &&                          \
        (OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2))
 

```

Reply:
It seems this will make `checkpatch` fail with `WARNING:LEADING_SPACE`.

Reply:
Sorry, I should've put tab in the sample code as an initial indent when copying. This is the format that `clang-format` suggests.

Reply:
Thanks, fixed.